src/modules/75-cf/pages/target-management/targets/__tests__/TargetsPage.test.tsx

```diff
@@ -201,15 +178,212 @@ describe('TargetsPage', () => {
       })
     })
 
-    const { container } = render(
-      <TestWrapper
-        path="/account/:accountId/cf/orgs/:orgIdentifier/projects/:projectIdentifier/feature-flags"
-        pathParams={{ accountId: 'dummy', orgIdentifier: 'dummy', projectIdentifier: 'dummy' }}
-      >
-        <TargetsPage />
-      </TestWrapper>
-    )
+    const { container } = renderComponent()
 
     expect(container).toMatchSnapshot()
   })
+
+  test('TargetsPage should render a row of target', async () => {
+    renderComponent()
+    expect(document.querySelector('[class="TableV2--body"]')).toBeVisible()
```

Comment:
Your selector here is a bit too qualified. `querySelector` works like a CSS selector, so you can simplify.
```suggestion
    expect(document.querySelector('.TableV2--body')).toBeVisible()
```

Reply:
As-is that would then break if the table got a second class.

Reply:
updated